xwiki-platform-core/pom.xml

```diff
@@ -174,6 +174,19 @@
                 </item>
               </differences>
             </revapi.differences>
+            <revapi.differences>
+              <differences>
+                <item>
+                  <ignore>true</ignore>
+                  <code>java.annotation.added</code>
+                  <old>method void org.xwiki.rendering.macro.display.DisplayMacroParameters::setReference(java.lang.String)</old>
+                  <new>method void org.xwiki.rendering.macro.display.DisplayMacroParameters::setReference(java.lang.String)</new>
+                  <annotation>@org.xwiki.properties.annotation.PropertyMandatory</annotation>
```

Comment:
We may want to change the following instead: https://github.com/xwiki/xwiki-commons/blob/4da483054e412d315dce36d34c24fd5ef454ae59/xwiki-commons-tools/xwiki-commons-tool-verification-resources/src/main/resources/revapi.json#L59-L72

Do you see any case where adding this annotation would be a breakage?

Reply:
That's a good point. I think it has already been discussed but I don't recall the conclusion. I don't think it would hurt.